packages/admin-ui-form/package.json

```diff
@@ -5,7 +5,7 @@
   "version": "0.2.0",
   "main": "dist/vtex-admin-ui-form.cjs.js",
   "module": "dist/vtex-admin-ui-form.esm.js",
-  "typings": "dist/index.d.ts",
+  "typings": "dist/vtex-admin-ui-form.cjs.d.ts",
```

Comment:
With this, the typescript declarations work

Reply:
using `yarn preconstruct validate` you can check if the setup is right and using `yarn preconstruct fix` it will fix the issues

Reply:
Where should we document such a thing?